Source/Core/UpdaterCommon/UpdaterCommon.cpp

```diff
@@ -425,25 +424,25 @@ bool UpdateFiles(const std::vector<TodoList::UpdateOp>& to_update,
       std::string contents;
       if (!File::ReadFileToString(path, contents))
       {
-        LogToFile("Could not read existing file %s.\n", op.filename.c_str());
+        LogToFile("Could not read existing file %s.\n", filename.c_str());
         return false;
       }
       Manifest::Hash contents_hash = ComputeHash(contents);
-      if (contents_hash == op.new_hash)
+      if (contents_hash == new_hash)
       {
-        LogToFile("File %s was already up to date. Partial update?\n", op.filename.c_str());
+        LogToFile("File %s was already up to date. Partial update?\n", filename.c_str());
         continue;
       }
-      else if (!op.old_hash || contents_hash != *op.old_hash || is_self)
+      else if (!old_hash || contents_hash != *_old_hash || is_self)
```

Comment:
Shouldn't it be `old_hash` instead of `_old_hash` according to the buildbot error?

Reply:
Yes, thanks for catching that